include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -274,6 +327,13 @@ class MGTransferGlobalCoarsening : public dealii::MGTransferBase<VectorType>
     const MGLevelObject<MatrixType> &                         matrices,
     const MGLevelObject<MGTwoLevelTransfer<dim, VectorType>> &transfer);
 
+  /**
+   * TODO.
```

Comment:
You'll specify this comment and some other parts in #11699. So this PR is blocked by #11699?